pkg/standalone/standalone.go

```diff
@@ -448,10 +451,27 @@ func runPlacementService(wg *sync.WaitGroup, errorChan chan<- error, info initIn
 		errorChan <- fmt.Errorf("%s container exists or is running. %s", placementContainerName, errInstallTemplate)
 		return
 	}
+	var image string
+
+	imgInfo := daprImageInfo{
+		ghcrImageName:      daprGhcrImageName,
+		dockerHubImageName: daprDockerImageName,
+		imageRegistryURL:   info.imageRegistryURL,
+		imageRegistryName:  defaultImageRegistryName,
+	}
 
-	if info.fromDir != "" {
-		dir := path_filepath.Join(info.fromDir, dockerImageSubDir)
-		err = loadDocker(dir, image)
+	if isAirGapInit(info.fromDir) {
```

Comment:
As this check is getting used at various places in standalone.go, should we just set a boolean flag once and use it everywhere, instead of checking every time if fromDir's length is 0 after trimming space.

Reply:
+1